Update scoped styles documentation in guides (#11311)
withastro/docs: src/content/docs/en/guides/styling.mdx

<html><!-- Your page here --></html>
```

> If your package **does _not_ suggest using a file extension** (i.e. `package-name/styles`), you'll need to update your Astro config first!

Reviewer: ```suggestion
If your package **does not suggest using a file extension** (i.e. `package-name/styles`), you'll need to update your Astro config first!
```

- **imported styles**
- **scoped styles** (highest precedence)

### Scoped Styles

Depending on your chosen value for [`scopedStyleStrategy`](/en/reference/configuration-reference/#scopedstylestrategy), scoped styles may or may not increase the [CLASS column specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_cascade/Specificity#class_column).

> That being said, [scoped styles](#scoped-styles) will always come last in the order of appearance. These styles will therefore take precedence over other styles of the same specificity. For example, if you import a stylesheet that conflicts with a scoped style, the scoped style’s value will apply:

Reviewer: ```suggestion
However, [scoped styles](#scoped-styles) will always come last in the order of appearance. These styles will therefore take precedence over other styles of the same specificity. For example, if you import a stylesheet that conflicts with a scoped style, the scoped style’s value will apply:
```
This is more our language style, and simpler, more common words are easier to read (and to translate!)

</div>
```

> Scoped styles will be overwritten if the imported style is _more specific_. The style with a higher specificity will take precedence over the scoped style:

Reviewer: ```suggestion
Scoped styles will be overwritten if the imported style is more specific. The style with a higher specificity will take precedence over the scoped style:
```
Just also taking the opportunity to remove italics from older docs when we come across them, as it's no longer our style to use them!
